pkgs/development/python-modules/aioshelly/default.nix

```diff
@@ -7,13 +7,13 @@
 
 buildPythonPackage rec {
   pname = "aioshelly";
-  version = "0.6.4";
+  version = "1.0.0";
 
   src = fetchFromGitHub {
     owner = "home-assistant-libs";
     repo = pname;
     rev = version;
-    sha256 = "sha256-QRCqkaKhPQQjNt9mw8nlTB5YKLmIZbXfrxarb3Ksr5k=";
+    sha256 = "sha256-6CN2bY/dxOqCvL7a8BpVUOu5WL1WKMs9vK5q27Tvaqc=";
```

Comment:
`sha256 =` could be replaced with `hash =` here, which I believe is preferable (https://nixos.wiki/wiki/Nix_Hash#Usage).